### DISCUSSION TITLE
rolling restart in Trino

### DISCUSSION THREAD

**Discussion by natarajank11** [trinodb/trino#17675] [General]
we use Trino 407. whenever there is some UDF functionality to be deployed, we need to do rolling restart without affecting production down.
As far as i know, Trino restarts all the nodes in the cluster. Is there any way to support rolling restart?

**Comment by hashhar** [comment]
https://github.com/trinodb/trino/discussions/17515

**Reply by hashhar** [reply]
Other than that there's no way to reload plugins without restarting nodes. You can restart one worker at a time as mentioned in the link.
